### ISSUE TITLE
[Feature] Add `Connecting` status to VPN block

### ISSUE THREAD

**Issue by totikom** [greshake/i3status-rust#1882]
Right now vpn block has only 3 statuses: `Connected`, `Disconnected` and `Error`.
The way drivers work now leads to "connecting" status being displayed as `Error`.
I can implement it, but I don't have a NordVPN account, so I'll need help with testing of `NordVpnDriver`.

**Comment by MaxVerevkin** [comment]
Is this necessary for #1877?

**Comment by totikom** [comment]
No, it will be an addition for both NordVPN and Mullvad drivers.


I think, that it will be better to report `connecting` as a separate status (now everything except `connected` and `disconnected` is shown as `error`)

**Comment by totikom** [comment]
I've started working on this feature.
